modules/multi-architecture-modify-machine-set-gcp.adoc

```diff
@@ -121,27 +124,35 @@ Use the `project` and `name` parameters from the output to create the path to im
 $ projects/<project>/global/images/<image_name>
 ----
 <4> Optional: Specify custom metadata in the form of a `key:value` pair. For example use cases, see the GCP documentation for link:https://cloud.google.com/compute/docs/metadata/setting-custom-metadata[setting custom metadata].
-<5> Specify an ARM64 supported machine type. For more information, refer to _Tested instance types for GCP on 64-bit ARM infrastructures_ in "Additional resources".
+<5> Specify a 64-bit ARM supported machine type. For more information, see "Tested instance types for GCP on 64-bit ARM infrastructures".
```

Comment:
Same issue, can we generalize and discuss about a secondary architecture AND higlighting that instance/machine type and RHCOS image should be for the same arch?